app/models/grda_warehouse/hmis_form.rb

```diff
@@ -500,6 +559,29 @@ def <=>(other)
   end
 
   def self.rrh_assessment_name
-    'Boston CoC Coordinated Entry Assessment'
+    GrdaWarehouse::HMIS::Assessment.rrh_assessment&.first&.name
   end
+
+  def section_starts_with(string)
+      relevant_section = answers[:sections].select do |section|
+        section[:section_title].downcase.starts_with?(string.downcase)
+      end&.first
+      return false unless relevant_section.present?
+
+      relevant_section
+    end
+
+    #
+    # Finds the first relevant answer where the question includes the string.
+    #
+    # @param [Hash] section part of the HmisForm answers column
```

Comment:
I like to include the parameter names, but that may just be my JavaDoc showing...